Code upgrade for TensorFlow 2

madgagarin opened the pull request:
@googlebot I signed it!

madgagarin committed:
Code upgrade for TensorFlow 2.0

madgagarin commented:
@googlebot I signed it!

LowSkillDeveloper commented:
also should change 

`tf.gfile.`

to 

`tf.io.gfile.`

LowSkillDeveloper commented:
And need replace 
```
import tensorflow as tf
```
to
```
import tensorflow.compat.v1 as tf
tf.disable_v2_behavior()
```
in the label_image.py

tiandiao123 commented:
Thank you for noticing!!!

duanegran committed:
Update label_image.py

Running this with TensorFlow 2.4 on a Raspberry Pi I was getting a "RuntimeError: The Session graph is empty" notice and the label operation wouldn't work.  I tried various things to fix it, but the only thing that seemed to work was disabling eager execution.  I'm not an expert with TensorFlow so I'm open to other solutions if this is a problem.  Just sharing that in my case it resolved the problem and allowed image classification to work again.

madgagarin committed:
Merge pull request #1 from duanegran/patch-1

Update label_image.py

madgagarin committed:
Update retrain.py

madgagarin committed:
up to 2.4

madgagarin committed:
up to 2.4

madgagarin committed:
up to 2.4
